fix a bug when features are enabled "Notice: Undefined property: stdClass::$i18n_mode in i18n_taxonomy_form_taxonomy_form_vocabulary_alter()"
=========================================================================================================================
--- sites/all/modules/contributed/i18n/i18n_taxonomy/i18n_taxonomy.module	2012-01-14 00:12:30.000000000 +0100
+++ sites/all/modules/contributed/i18n/i18n_taxonomy/i18n_taxonomy.module	2012-02-21 10:44:29.000000000 +0100
@@ -597,7 +597,7 @@
  */
 function i18n_taxonomy_taxonomy_vocabulary_update($vocabulary) {
   // Update language for related terms
-  switch ($vocabulary->i18n_mode) {
+  switch (i18n_taxonomy_vocabulary_mode($vocabulary)) {
     case I18N_MODE_LANGUAGE:
       $update['language'] = $vocabulary->language;
       break;
	   